WGS_pipeline/Aggregate_CN_seg_files.wdl

```diff
@@ -1,34 +0,0 @@
-# Given a set of samples, combine segment files into a single file
```

Comment:
could we move it offline now?

Reply:
otherwise it is not needed there to still exist on terra (I think).

Reply:
I'm not sure if I have time to test it before the release... so just to be safe let's keep it for now? Since the terra workflow imports from dockstore, we do need it on master for it to work

Reply:
I mean. here is the code:

```python
res = pd.concat([file for file in wm.get_samples().loc[wm.get_sample_set()['samples'].tolist(), expression_col].tolist()])
```
it is really one line.